spec/opensearch_spec_helper.rb

```diff
@@ -1,14 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+#  The OpenSearch Contributors require contributions made to
+#  this file be licensed under the Apache-2.0 license or a
+#  compatible open source license.
+#
+#  Modifications Copyright OpenSearch Contributors. See
+#  GitHub history for details.
+
 require_relative './spec_helper'
 
 require 'elasticsearch'
```

Comment:
We still need elasticsearch in this file?

Reply:
For integration tests, we are using elasticsearch ruby api to create,delete,update index.